src/options/Main/OptionsList/ThemeSelection.jsx

```diff
@@ -0,0 +1,39 @@
+import { h } from 'preact';
+
+// import { Component, h } from 'preact';
```

Comment:
If we don't need this, it can be deleted

Reply:
👍